backend/Dockerfile

```diff
@@ -39,4 +25,4 @@ HEALTHCHECK --interval=30s --timeout=3s CMD curl -f http://localhost/:3000
 USER app
 
 # Start up command with 50MB of heap size, each application needs to determine what is the best value. DONT use default as it is 4GB.
-ENTRYPOINT ["node","--max-old-space-size=50", "dist/main"]
+CMD ["/app/dist/main"]
```

Comment:
if we are going from entrypoint to command, may be we should add the old space limit for Garbage collection as env var
export NODE_OPTIONS="--max-old-space-size=50"

Reply:
Good catch and done!  :)